keps/sig-network/4444-service-traffic-distribution/README.md

```diff
@@ -266,6 +267,18 @@ NOTE: Implementations reserve the right to refine the behavior associated with
   implementation. This simplifies their deployment process and reduces the
   complexity of managing cross-cluster applications.
 
+#### Story 4 (DNS)
+
+* **Requirement:** As a cluster administrator, I plan to deploy CoreDNS so that there
```

Comment:
yeah, this is common, to have the DNS resolver per node ... this also removes the nodeLocalDNS hack to divert the traffic, and people/installers: kops, kubeadm, ...  can just replace the DNS deployment by a daemonset